blog/docs/articles/demystifying-activator-on-path.md

```diff
@@ -0,0 +1,325 @@
+It is defined as:
+
+```
+EBC = TotalCapacity - ObservedPanicValue - TargetBurstCapacity(TBC).
+```
+
+`TotalCapacity` is calculated as ready_pod_count*revision_total. The default TBC is set to 200. 
+Now with the above defaults and given the fact that a request needs to stay around for some time in order concurrency metrics to show enough load, it means you can't get EBC>=0 with a hello-world example.
+The latter which is often confusing for the newcomer as the Knative service never enters serve mode. 
+In the next example we will show the lifecycle of a Knative service that also moves to serve mode and how ebc is calculated in practice. 
+
+Here is a service that has a target concurrency of 10 and tbc=10:
+
+```
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: autoscale-go
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/target: "10"
+        autoscaling.knative.dev/target-burst-capacity: "10"
+    spec:
+      containers:
+      - image: ghcr.io/knative/autoscale-go:latest
+```
+
+The scenario we are going to demonstrate is deploye the ksvc, let it scale down to zero and then send traffic for 10 minutes.
+We then collect the logs from the autoscaler and visualize the EBC, ready pods and panic mode over time.
+The graphs are shown next.
+
+![Excess burst capacity over time](/blog/articles/images/ebc.png)
+![Ready pods over time](/blog/articles/images/readypods.png)
+![Panic mode over time](/blog/articles/images/panic.png)
+
+
+Let's describe in detail what we see above. Initially when the ksvc is deployed there is no traffic and one pod is created by default for verification reasons.
+
+Until the pod is up we have:
+```bash
+$ kubectl get sks
+NAME                 MODE    ACTIVATORS   SERVICENAME          PRIVATESERVICENAME           READY     REASON
+autoscale-go-00001   Proxy   2            autoscale-go-00001   autoscale-go-00001-private   Unknown   NoHealthyBackends
+```
+When the pod is up we have:
+
+```bash
+$ oc get po
+NAME                                             READY   STATUS    RESTARTS   AGE
+autoscale-go-00001-deployment-6cc679b9d6-xgrkf   2/2     Running   0          24s
+
+$ oc get sks
+NAME                 MODE    ACTIVATORS   SERVICENAME          PRIVATESERVICENAME           READY   REASON
+autoscale-go-00001   Serve   2            autoscale-go-00001   autoscale-go-00001-private   True    
+```
+
+The reason why we are in Serve mode is that because EBC=0. When you enable debug logs, in the logs you get:
+
+```
+  "severity": "DEBUG",
+  "timestamp": "2023-10-10T15:29:37.241575214Z",
+  "logger": "autoscaler",
+  "caller": "scaling/autoscaler.go:286",
+  "message": "PodCount=1 Total1PodCapacity=10.000 ObsStableValue=0.000 ObsPanicValue=0.000 TargetBC=10.000 ExcessBC=0.000",
+  "commit": "f1617ef",
+  "knative.dev/key": "default/autoscale-go-00001"
```

Comment:
may want to consider filtering out parts here that aren't relevant to what you want to show, as they might obscure the info you want to highlight to the reader

Reply:
I cleaned up the log msgs.